assets/src/dashboard/app/views/templateDetails/content/relatedGrid/index.js

```diff
@@ -38,7 +39,10 @@ function RelatedGrid({ pageSize, relatedTemplates, templateActions }) {
 
   return (
     <RowContainer>
-      <SubHeading>{__('Related Templates', 'web-stories')}</SubHeading>
+      <Headline size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL} as="h3">
+        {__('Related Templates', 'web-stories')}
+      </Headline>
+
```

Comment:
Looks like this needs a little more space

|Current|Designs|
|--|--|
|![image](https://user-images.githubusercontent.com/22185279/110033286-731cd300-7cf6-11eb-8cbc-403bfd27d91b.png)|![image](https://user-images.githubusercontent.com/22185279/110033338-7f089500-7cf6-11eb-9665-55bb04ce79a9.png)|

Reply:
yup 🤦  updated the element to an h3 and it lost its style inheritance.
![Screen Shot 2021-03-04 at 3 09 28 PM](https://user-images.githubusercontent.com/10720454/110037472-ae6dd080-7cfb-11eb-9f8c-b73cdf6f0d5b.png)